coveo_stew/environment.py

```diff
@@ -127,3 +130,22 @@ def find_python_tool(
 - Or you can install {tool} to your system so that it can be found in the PATH
 """
     )
+
+
+RE_POETRY_VERSION: Pattern = re.compile(r"version (?P<version>\d\.\d\.\d)")
```

Comment:
:thinking: ?
```suggestion
RE_POETRY_VERSION: Pattern = re.compile(r"version (?P<version>\d+\.\d+\.\d+)")
```

Should we make this case insensitive also to make the regex more robust to changes? (for instance, `version` -> `Version`)